.circleci/config.yml

```diff
@@ -61,11 +61,9 @@ jobs:
       - attach_workspace:
           at: /tmp/workspace
       - setup_remote_docker
-      - run: docker pull gcr.io/projectriff/streaming-http-adapter:0.1.3
-      - run: docker pull gcr.io/projectriff/node-function:0.6.1
       - run: docker load < $(ls /tmp/workspace/pack-*-build.tar | head -1)
       - run: docker load < $(ls /tmp/workspace/pack-*-run.tar | head -1)
-      - run: pack create-builder << parameters.image_tag >> --config << parameters.builder_toml >> --pull-policy never
+      - run: pack create-builder << parameters.image_tag >> --config << parameters.builder_toml >> --pull-policy if-not-present
```

Comment:
I _think_ I'm ok with this now. Originally `pack` only support `never` via `--no-pull`. We needed to not `pull`, otherwise the act of creating a builder would use the published build/run images and not use the ones built in the CI run.

I wish `pack` had a pull policy for buildpacks vs image pull policies. If we make a mistake and `docker load` gets removed or mistyped we could be publishing builders on old images.

Reply:
Thanks for the explanation behind `never`, I was wondering why we'd do it this way. Makes sense. If we want to avoid this potential issue, we can do smth. like this before calling `create-builder`:

```bash
#!/usr/bin/env bash

for buildpack_uri in $(yj -t <builder-18.toml | jq -r '.buildpacks[].uri | select(startswith("docker://"))'); do
    docker pull "${buildpack_uri}"
done
```

With this, we can keep `never` but don't have to maintain two lists of images to pull. What do you think @jabrown85?

Reply:
I think I'm ok leaving it as you have it right now. I like the little bash helper, but it won't help buildpacks.io registry published images.